### ISSUE TITLE
Shift Tab won't close datepicker

### ISSUE THREAD

**Issue by humphries40** [airbnb/react-dates#695]
I've noticed that if you click on a datepicker, then shift tab out of the datepicker it stays open. This was repeatable on Storybook and in my solution.   Here's a GIF of it. 
![shift tabbing issue](https://user-images.githubusercontent.com/2933741/29837979-cd9960d2-8cc7-11e7-879f-3224ffff1804.gif)

**Comment by majapw** [comment]
Oh that's so weird. If you tab forward and then tab back, it seems to close, but that doesn't seem to take on the first click. Def a bug.

**Comment by humphries40** [comment]
Yeah, definitely weird. I tried a bunch of different things that the tabbing worked fine, but then this one little way it doesn't.

**Comment by inspiratweb** [comment]
I think is the same bug. If you are navigating through days with arrow keyboard and you `tab`... next field is selected but the picker is not closed. Clicking out works fine
![ezgif-3-736432db24](https://user-images.githubusercontent.com/6814061/30640681-2032bd3c-9e04-11e7-9f0e-37092243f277.gif)

**Comment by VinceVachon** [comment]
Any updates?

**Comment by mmarkelov** [comment]
Should be fixed by #1741
